wordpress_rock/rockcraft.yaml

```diff
@@ -0,0 +1,188 @@
+          unzip "${plugin}.zip"
+          rm "${plugin}.zip"
+      done
+      curl -sSL "https://downloads.wordpress.org/plugin/openid.3.5.0.zip" -o "openid.zip"
+      unzip "openid.zip"
+      rm "openid.zip"
+      # Latest YoastSEO does not support 5.9.3 version of WordPress.
+      curl -sSL "https://downloads.wordpress.org/plugin/wordpress-seo.18.9.zip" -o "wordpress-seo.zip"
+      unzip "wordpress-seo.zip"
+      rm "wordpress-seo.zip"
+      git clone https://git.launchpad.net/~canonical-sysadmins/wordpress-launchpad-integration/+git/wordpress-launchpad-integration wordpress-launchpad-integration
+      git clone https://git.launchpad.net/~canonical-sysadmins/wordpress/+git/openstack-objectstorage-k8s openstack-objectstorage-k8s
+      git clone https://git.launchpad.net/~canonical-sysadmins/wordpress-teams-integration/+git/wordpress-teams-integration wordpress-teams-integration
+      git clone https://git.launchpad.net/~canonical-sysadmins/wordpress/+git/wp-plugin-xubuntu-team-members xubuntu-team-members
+      rm -rf */.git)
+
+      # Install wordpress themes
+      (cd wordpress_install_dir/wp-content/themes;
+      git clone https://git.launchpad.net/~canonical-sysadmins/ubuntu-community-webthemes/+git/light-wordpress-theme light-wordpress-theme
+      git clone https://git.launchpad.net/~canonical-sysadmins/wordpress/+git/wp-theme-mscom mscom
+      git clone https://git.launchpad.net/~canonical-sysadmins/wordpress/+git/wp-theme-thematic thematic
+      git clone https://git.launchpad.net/~canonical-sysadmins/wordpress/+git/wp-theme-twentyeleven twentyeleven
+      git clone https://git.launchpad.net/~canonical-sysadmins/ubuntu-cloud-website/+git/ubuntu-cloud-website ubuntu-cloud-website
+      git clone https://git.launchpad.net/~canonical-sysadmins/wordpress/+git/wp-theme-ubuntu-community ubuntu-community
+      git clone https://git.launchpad.net/~canonical-sysadmins/ubuntu-community-wordpress-theme/+git/ubuntu-community-wordpress-theme ubuntu-community-wordpress-theme
+      git clone https://git.launchpad.net/~canonical-sysadmins/wordpress/+git/wp-theme-ubuntu-fi ubuntu-fi
+      git clone https://git.launchpad.net/~canonical-sysadmins/wordpress/+git/wp-theme-ubuntu-light ubuntu-light
+      git clone https://git.launchpad.net/~canonical-sysadmins/wordpress/+git/wp-theme-ubuntustudio-wp ubuntustudio-wp
+      git clone https://git.launchpad.net/~canonical-sysadmins/wordpress/+git/wp-theme-launchpad launchpad
+      git clone https://git.launchpad.net/~canonical-sysadmins/wordpress/+git/wp-theme-xubuntu-website xubuntu-website
+      bzr branch lp:resource-centre;
+      rm -rf */.git)
+
+      cp -R . $CRAFT_PART_INSTALL
+    organize:
+      wordpress_install_dir: /var/www/html
+      wp: /usr/local/bin/wp
+    override-prime: |
+      craftctl default
+      chown $IMAGE_RUN_USER_ID:$IMAGE_RUN_GROUP_ID -R --no-dereference "$CRAFT_PRIME/var/www/html"
```

Comment:
If you use multiple parts, you will need to have a step after all the others to change the ownership (like [here](https://github.com/canonical/discourse-k8s-operator/blob/main/discourse_rock/rockcraft.yaml#L193))